controller/workitemtype.go

```diff
@@ -120,12 +120,7 @@ func (c *WorkitemtypeController) List(ctx *app.ListWorkitemtypeContext) error {
 			return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, "Error listing work item types"))
 		}
 		// Remove "planneritem" from the list of WITs
-		witModels := []workitem.WorkItemType{}
-		for _, wit := range witModelsOrig {
-			if wit.ID != workitem.SystemPlannerItem {
-				witModels = append(witModels, wit)
-			}
-		}
+		witModels := stripBaseWorkItemTypes(witModelsOrig)
```

Comment:
why do you want to exclude the base type at the controller level instead of the repository level ? Doing it at the repository level means that any controller calling the `ListPlannerItems()` method will benefit from the exclusion.